test/common/dailies.coffee

```diff
@@ -49,9 +50,15 @@ newUser = (addTasks=true)->
       user.ops.addTask {body: {type: task, id: shared.uuid()}}
   user
 
+getTasksForUser = (user) ->
+  return user.habits.concat(user.dailys)
+          .concat(user.todos).concat(user.rewards)
+
 cron = (usr, missedDays=1) ->
   usr.lastCron = moment().subtract(missedDays,'days')
-  usr.fns.cron()
+  daysMissed = usr.fns.shouldCronRun()
+  if daysMissed isnt 0 
```

Comment:
Well, the good news is that it is working as intended, so it must be something wrong with how the test is written. I'm guessing that since the object is no longer a part of the user, the mock daily object isn't actually being updated.